tests/subsys/net/lib/lwm2m_client_utils/CMakeLists.txt

```diff
@@ -46,6 +46,16 @@ set(options
   -DCONFIG_LWM2M_CLIENT_UTILS_RAI
   -DCONFIG_LWM2M_ENGINE_DEFAULT_LIFETIME=43200
   -DCONFIG_LWM2M_CLIENT_UTILS_DTLS_CON_MANAGEMENT
+  -DCONFIG_LTE_LC_CONEVAL=1
+  -DCONFIG_LTE_LC_EDRX=1
+  -DCONFIG_LTE_LC_NCELLMEAS=1
+  -DCONFIG_LTE_LC_PERIODICSEARCHCONF=1
```

Comment:
Maybe just add the ones this needs. I'd assume `CONFIG_LTE_LC_PERIODICSEARCHCONF` and many others are not needed?
This way we would also do a bit of testing for other than default or all modules configurations.